openstack/nova/values.yaml

```diff
@@ -65,6 +65,12 @@ defaults:
         resume_guests_state_on_host_boot: true
         # enables collecting metrics for RPC calls
         statsd_enabled: true
+      config_file:
+        libvirt:
+          volume_use_multipath: True
+          connection_uri: "qemu:///system"
+        os_vif_ovs:
+          ovsdb_connection: "unix:/run/openvswitch/db.sock"
```

Comment:
Are these config options used inside of the nova chart? If yes, could you shortly describe for what?

Reply:
Sorry, left-over from testing. I've removed the whole KVM section